examples/main.tf

```diff
@@ -5,7 +5,7 @@ data "aws_kms_key" "sns" {
 module "sqs_queue" {
```

Comment:
Can you create a folder for the default example>

Reply:
done